util.go

```diff
@@ -171,3 +178,49 @@ func checkFormatInt64(typeName string, schemaObject *SchemaObject) {
 		schemaObject.Format = "int64"
 	}
 }
+
+func checkAndMutatePackageName(pkgName string, parser *parser) string {
```

Comment:
Probably don't want to be mutating an input, especially when we don't need to - the calling code is relying on the returned result, shouldn't we generate a new string to be returned instead?